spec/services/disability_claim_service_spec.rb

```diff
@@ -3,8 +3,8 @@
 
 RSpec.describe DisabilityClaimService do
   let(:user) { User.sample_claimant }
-  let(:claim) do
-    FactoryGirl.create(:disability_claim, user_uuid: user.uuid, data: {})
+  let!(:claim) do
+    FactoryGirl.create(:disability_claim, user_uuid: user.uuid)
```

Comment:
Unless we go down the route of stubbing `find` 😉 
The behavior being tested here is that if the API call fails due to EVSS being down, the data is retrieved from the database which contains the last known state of the claim. We can use a `before(:all)` to set up the database, but would need to also manually clean up in an `after(:all)` and reload from the db again for each spec as suggested here https://relishapp.com/rspec/rspec-rails/docs/transactions 
I don't think you'd get much savings in performance from this (persist once, then reload for each test) vs. persisting a claim for each test